src/FffGcodeWriter.cpp

```diff
@@ -176,18 +173,18 @@ void FffGcodeWriter::initConfigs(SliceDataStorage& storage)
 
 void FffGcodeWriter::processStartingCode(SliceDataStorage& storage)
 {
-    if (!CommandSocket::isInstantiated())
-    {
-        std::ostringstream prefix;
-        prefix << "FLAVOR:" << toString(gcode.getFlavor());
-        gcode.writeComment(prefix.str().c_str());
-        if (gcode.getFlavor() == EGCodeFlavor::ULTIGCODE)
-        {
-            gcode.writeComment("TIME:666");
-            gcode.writeComment("MATERIAL:666");
-            gcode.writeComment("MATERIAL2:-1");
-        }
-    }
+    //if (!CommandSocket::isInstantiated())
+    //{
+    //    std::ostringstream prefix;
+    //    prefix << "FLAVOR:" << toString(gcode.getFlavor());
+    //    gcode.writeComment(prefix.str().c_str());
+    //    if (gcode.getFlavor() == EGCodeFlavor::ULTIGCODE)
+    //    {
+    //        gcode.writeComment("TIME:666");
+    //        gcode.writeComment("MATERIAL:666");
+    //        gcode.writeComment("MATERIAL2:-1");
+    //    }
+    //}
```

Comment:
I tried to make the code independent whether a commandSocket instantiated or not.
Is it ok if it runs in the instantiated case too?